bc_obps/compliance/tests/service/bc_carbon_registry/test_apply_compliance_units_service.py

```diff
@@ -476,6 +478,8 @@ def test_apply_compliance_units_success(
         mock_invoice.elicensing_line_items.all.return_value = []
         mock_refresh_data.return_value.invoice = mock_invoice
 
+        mock_bccr_service.validate_holding_account_ownership.return_value = True
```

Comment:
It would be a good test to assert that `validate_holding_account_ownership` is called after applying units.